src/richie/apps/core/serializers.py

```diff
@@ -8,12 +9,23 @@ class UserSerializer(ModelSerializer):
     """Serializer for User objects."""
 
     full_name = SerializerMethodField()
+    links = SerializerMethodField()
 
     class Meta:
         model = get_user_model()
-        fields = ["full_name", "username"]
+        fields = ["full_name", "links", "username"]
 
     @staticmethod
     def get_full_name(user):
         """Get the user's full_name on the User class during serialization."""
         return user.get_full_name()
+
+    @staticmethod
+    def get_links(user):
+        """Get the user's links related to LMS"""
+        # TODO Make things resilient to MULTI LMS
```

Comment:
In order to do multi-LMS, these links will have to point on a dashboard in Richie, and this "links" property can most probably be removed :crossed_fingers: